sphinx_immaterial/apidoc/object_toc.py

```diff
@@ -294,7 +294,14 @@ def depart_term(
     node: docutils.nodes.term,
     super_func: html_translator_mixin.BaseVisitCallback[docutils.nodes.term],
 ) -> None:
-    self.add_permalink_ref(node, _("Permalink to this definition"))
+    next_node: docutils.nodes.Element = node.next_node(descend=False, siblings=True)
+    if (
+        not isinstance(next_node, docutils.nodes.classifier)
+        and node.parent is not None
```

Comment:
This should be `and not (node.parent is not None and node.parent.parent is not None and isinstance(node.parent.parent.parent, sphinx.addnodes.glossary))`

Reply:
done